### PR TITLE
4.2.0 beta blog

### PR DESCRIPTION
**Pull request by JohnMarksRH**
Changes proposed in this pull request:

- New post about 4.2.0 beta 

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @jmarks

This pull request needs review by: @mykaul

### CODE REVIEW

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Introducing oVirt 4.2.0 Beta
+author: jmarks
+tags: ovirt, 42.0
```
Comment (mykaul) [inline source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: 42.0 is the right tag?

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+oVirt 4.2.0 Beta features **Gluster 3.12**.
+
+
+oVirt's hyperconverged solution now enables a **single replica Gluster deployment**.
+
+
+**OVN (Open Virtual Network)** is now fully supported and recommended for isolated overlay networks. OVN is automatically deployed on the the host, and made available for VM connectivity.
+
+
+**Snapshots** - such as Python SDKs - can now be uploaded and downloaded.
```
Comment (mykaul) [inline source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: "such as Python SDKs - can now be uploaded and downloaded." -> "can now be uploaded and downloaded via the REST API (and the SDKs)

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+Improvements have been introduced to **eliminate a single point of failure (SPOF) in an iSCSI single target**. Now, the hosted-engine will connect to all IPs discovered, allowing both higher performance via multiple paths as well as high availability in the event that one of the targets fails.
```
Comment (mykaul) [inline source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: 1. Improvements have -> An improvement has
2. eliminate a single point of failure (SPOF) in an iSCSI single target**  -> for Hosted Engine

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+A new **self-hosted engine wizard UI** replaces the older version, and provides a greatly improved user experience. Among other improvements, it now enables users to review and edit their inputs before beginning the deployment process.
+
+
+**Windows DoD SPICE driver**, for seamless remote access to virtual machines. The driver aims to improve the user experience and performance for Windows graphical guests.
```
Comment (mykaul) [inline source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: I'm not sure what the 'DoD' here means actually...

Comment (JohnMarksRH) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: Display only Driver? I deleted it (red hat access docs refer to "Windows SPICE driver"

Comment (mykaul) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: Yes, Display Only Driver makes sense (DoD might seems like Department of Defense...)

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md (hunk 2 above)
Comment (JohnMarksRH) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: fixed

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md (hunk 1 above)
Comment (JohnMarksRH) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: fixed

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md (hunk 3 above)
Comment (JohnMarksRH) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: fixed

File: source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md (hunk 4 above)
Comment (JohnMarksRH) [reply source/blog/2017-11-02-introducing-ovirt-4.2.0-beta.html.md]: changed

**Pull request merged**